gis/mapserver/Portfile

```diff
@@ -110,7 +110,7 @@ foreach pyver ${python_suffixes} {
     set py_dot_ver [string index ${pyver} 0].[string range ${pyver} 1 end]
     set conf [lreplace ${python_variants} ${index} ${index}]
 
-    variant python${pyver} description "Add Python MapScript for python${pyver}" conflicts {*}${conf} "
+    variant python${pyver} description "Add Python MapScript for Python ${py_dot_ver}" conflicts {*}${conf} "
```

Comment:
can `312` added as a (default) Python variant?

Reply:
Thanks for reminding me, added Python 3.12 variant. (Python API is not mandatory, so stays optional only).